Source/JavaScriptCore/API/JSAPIGlobalObject.mm

```diff
@@ -279,7 +279,7 @@
     VM& vm = this->vm();
     auto scope = DECLARE_THROW_SCOPE(vm);
 
-    Identifier key = Identifier::fromString(vm, String { [[script sourceURL] absoluteString] });
+    Identifier key = Identifier::fromString(vm, String { [script sourceURL].absoluteString });
```

Comment:
I just briefly scrolled through the PR, and do not intend on doing a full review, but the machine output definitely looks like it needs another cleanup pass.